graveler/ref/manager.go

```diff
@@ -72,6 +72,19 @@ func (m *Manager) CreateRepository(ctx context.Context, repositoryID graveler.Re
 	return err
 }
 
+func (m *Manager) CreateBareRepository(ctx context.Context, repositoryID graveler.RepositoryID, repository graveler.Repository) error {
+	_, err := m.db.Transact(func(tx db.Tx) (interface{}, error) {
+		_, err := tx.Exec(
```

Comment:
This repeats code from `CreateRepository`.  That makes me want to keep it as a separate function even less.

Reply:
I guess it should be a composition.